pkg/services/authz/zanzana/zanzana.go

```diff
@@ -0,0 +1,60 @@
+package zanzana
+
+import (
+	"fmt"
+	"strconv"
+
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+)
+
+const (
+	TypeUser string = "user"
+	TypeTeam string = "team"
+)
+
+func NewObject(typ, id string) string {
+	return fmt.Sprintf("%s:%s", typ, id)
+}
+
+func NewScopedObject(typ, id, scope string) string {
+	return NewObject(typ, fmt.Sprintf("%s-%s", scope, id))
+}
+
+// rbac action to relation translation
+var actionTranslations = map[string]string{}
+
+type kindTranslation struct {
+	typ       string
+	orgScoped bool
+}
+
+// all kinds that we can translate into a openFGA object
+var kindTranslations = map[string]kindTranslation{}
```

Comment:
Is it for the future work to fill this structure?

Reply:
Yes it is. I extracted it from the other branch where I started a bit with dashboard and folders. But we will for sure need more properties to translate correctly :P